modules/system/helpers/Migration.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace System\Helpers;
+
+use Winter\Storm\Database\Model;
+
+/**
+ * This helper class is used in migration scaffolding console scripts
+ *
+ */
+class Migration
```

Comment:
I'm not sold on the idea of a helper class for this, at least not without me putting some more thought into the API I'd like to see for it, so can we just make this a protected method of the console command for now and then make the test look at the actual generated output of the console command?

Reply:
Dude, you're the one that specifically asked me to create a helper class for this...

ref. https://github.com/wintercms/winter/pull/935#discussion_r1252204195

Reply:
🤦‍♂️ welp I guess that proves my memory is shot 😂 

The original intention was to have a helper class that could be used to translate between forms, columns, migrations, etc for a given form/column/migration schema but I realized that was a bit overly ambitious for now so I wanted to switch back to a protected method so that we could get this PR merged without having to nail the public API the first time. Apologies for my confusion; hopefully that explanation makes sense and you agree - if not let me know.